addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -183,16 +194,25 @@
             </div>
         </div>
     </t>
-    <t t-name="wysiwyg.widgets.files.existing.content">
+    <t t-name="wysiwyg.widgets.image.existing.attachments">
+        <div class="existing-attachments">
+            <div class="row mt16 ml-auto" t-as="row" t-foreach="rows">
+                <div class="col-2 o_existing_attachment_cell" t-as="attachment" t-foreach="row">
+                    <i t-if="attachment.res_model === 'ir.ui.view'" class="fa fa-trash o_existing_attachment_remove" title="This file is a public view attachment" role="img" aria-label="This file is a public view attachment" t-att-data-id="attachment.id"/>
```

Comment:
I missed all of this, why duplicating the whole stuff here ? The only difference is the attachment name at the end ?

Reply:
On the following PR the image view is changed and not the document view is kept more or less the same.

The split has to be done, it's just a matter of which commit.

Reply:
We'll see then I guess but here it is one template for images and one for documents where documents only extends the image one :)

Reply:
done